### PR TITLE
Handled PLR DAO Membership case

### PR DESCRIPTION
**Pull request by parag-chauhan**
## Description
Handled PLR DAO Membership case with Preselect PLR token.

## Screenshots:
image

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### CODE REVIEW

**Review by Yash-Chaubal — APPROVED** [review]
LGTM!

File: src/components/TransactionBlock/PlrDaoStakingTransactionBlock.tsx [hunk 1]
```diff
@@ -214,7 +215,31 @@ const PlrDaoStakingTransactionBlock = ({
...
+    try {
+      const [providerAddressNfts, accountAddressNfts] = await Promise.all([
+        getNftsForChainId(CHAIN_ID.POLYGON, providerAddress, true),
+        getNftsForChainId(CHAIN_ID.POLYGON, accountAddress, true),
+      ]);
+      const nftCollection = [...providerAddressNfts, ...accountAddressNfts];
+      setIsNFTMember(
+        nftCollection.some((nft) => addressesEqual(nft.contractAddress, plrDaoMemberNft[STAKING_CHAIN_ID].address))
+      );
+    } catch (error) {
```
Comment (MayukhSamadder2000) [inline src/components/TransactionBlock/PlrDaoStakingTransactionBlock.tsx:231]: No need of error just do catch {}

File: src/components/TransactionBlock/PlrDaoStakingTransactionBlock.tsx [hunk 2]
```diff
@@ -214,7 +215,31 @@ const PlrDaoStakingTransactionBlock = ({
...
+      //
+    }
+  }, [accountAddress, providerAddress, sdk]);
+
+  useEffect(() => {
+    // Fetch a list of NFTs for the account to check if the user is existing member of PLR Dao.
+    getNftList();
+  }, [getNftList]);
```
Comment (MayukhSamadder2000) [inline src/components/TransactionBlock/PlrDaoStakingTransactionBlock.tsx:239]: Do we need to observe getNftList ?

**Review by MayukhSamadder2000 — APPROVED** [review]
Lgtm just a couple of comments

File: src/components/TransactionBlock/PlrDaoStakingTransactionBlock.tsx (hunk 2 above)
Comment (parag-chauhan) [reply src/components/TransactionBlock/PlrDaoStakingTransactionBlock.tsx:239]: Yes

**Pull request merged**